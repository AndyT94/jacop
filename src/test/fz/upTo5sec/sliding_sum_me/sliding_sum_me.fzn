var 3..3: low:: output_var;
var 7..7: up:: output_var;
array [1..7] of var int: variables:: output_array([1..7]) = [1,4,2,0,0,3,3];
solve  satisfy;
