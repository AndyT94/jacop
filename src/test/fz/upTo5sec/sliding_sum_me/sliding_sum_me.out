low = 3;
up = 7;
variables = array1d(1..7, [1, 4, 2, 0, 0, 3, 3]);
----------